### ISSUE TITLE
This needs an installer

### ISSUE THREAD

**Issue by dyeske** [jnielsendotnet/isboot#8]
Is this code from "peach" from about 10 years ago?  There was an installer of FreeBSD 9.x that could install over iscsi.  I'd like to see this become available again.  I work with ipxe a lot.

**Comment by jnielsendotnet** [comment]
Yes this is the same code. Writing an installer is out of my wheelhouse but I think you could use the standard install media. Boot up, enter a shell, start iscsid and connect the disk via iscsictl. Exit the shell, do the install, then enter a shell again and install isboot and make sure it loads at boot. Then reboot. Maybe all we need is for someone to try it and document the steps somewhere.

**Comment by dyeske** [comment]
I must fix pxe
